### PR TITLE
Refactor. Core build mode.

### PR DESCRIPTION
**Pull request by blckmn**

### CODE REVIEW

File: locales/en/messages.json [hunk 1]
```diff
@@ -6848,10 +6848,10 @@
...
     },
-    "classicBuild": {
-        "message": "Classic Mode"
+    "coreBuild": {
+        "message": "Core (formally 'classic')"
     },
-    "classicBuildModeDescription": {
-        "message": "This mode ignores all the parameters here. This essentially achieves the same as the classic nightlies, and is provided for convenience."
+    "coreBuildModeDescription": {
+        "message": "This mode loads the core hardware drivers (and some limited features). This produces a build that is the same as the builds released in the nightlies. It is provided here to assist in the detection of hardware on a flight controller, and is provided for that convenience only. There is no guarantee that all the features will be available (only hardware), for example GPS and LED_STRIP are not available on smaller flash sized targets like F722 and F411 using this mode."
```
Comment (haslinghuis) [inline locales/en/messages.json:6855]: This is likely to change soon, so better remove:

> for example GPS and LED_STRIP are not available on smaller flash sized targets like F722 and F411 using this mode.

**Comment by McGiverGim** [comment]
Are this rename necessary at this stage? Again, it produces that all the translators need to translate it again.

**Comment by blckmn** [comment]
> Are this rename necessary at this stage? Again, it produces that all the translators need to translate it again.

It's only renamed in english, the existing translations can remain - and if they get updated great, if not there is no harm done as "classic" is still a valid description.

File: locales/en/messages.json [hunk 2]
```diff
@@ -6848,10 +6848,10 @@
     "firmwareFlasherBuildCustomDefines": {
         "message": "Custom Defines"
     },
-    "classicBuild": {
-        "message": "Classic Mode"
+    "coreBuild": {
+        "message": "Core (formally 'classic')"
```
Comment (haslinghuis) [inline locales/en/messages.json:6852]: Core Mode ?

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: locales/en/messages.json (hunk 2 above)
Comment (blckmn) [reply locales/en/messages.json:6852]: I am not sure it is a "mode". If anything it would be `Core Only`

Comment (haslinghuis) [reply locales/en/messages.json:6852]: Please abandon term classic

**Review by haslinghuis — APPROVED** [review]

**Comment by McGiverGim** [comment]
> > Are this rename necessary at this stage? Again, it produces that all the translators need to translate it again.
> 
> It's only renamed in english, the existing translations can remain - and if they get updated great, if not there is no harm done as "classic" is still a valid description.

That does not work in this way. When you change the English translation, at Crowdin, all the translations are invalidated. Your work here modifying all the languages does not serve for nothing because it will be replaced with Crowdin files the next update.

**Review by chmelevskij — APPROVED** [review]

**Pull request merged**